### DISCUSSION TITLE
Is there any sample code for SP using node-saml?

### DISCUSSION THREAD

**Discussion by kg0r0** [node-saml/node-saml#42] [Q&A] [answered]
Is there any sample code for SP using node-saml?
The following document looks like a sample of passport-saml.
https://github.com/node-saml/node-saml/blob/master/docs/adfs/README.md

**Comment by cjbarth** [comment, marked as answer]
`node-saml` is the core of `passport-saml`, so they work similarly. There is currently no documentation specific to `node-saml`, but such documentation is welcome.

**Comment by markstos** [comment]
@kg0r0 Did you end up getting `node-saml` working without `passport-saml` or did you end using `passport-saml`?
